addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -350,6 +350,25 @@ $o-wslides-fs-side-width: 300px;
             }
         }
 
+        .o_wslides_slide_pulse {
+            animation: pulse .6s linear normal;
+        }
+
+        @keyframes pulse {
+            0%{
+                box-shadow: 0 0 0 0 rgba(23, 162, 184, .5), 0 0 0 0 rgba(23, 162, 184, .5);
+            }
+            40%{
+                box-shadow: 0 0 0 10px rgba(23, 162, 184, 0), 0 0 0 6px rgba(23, 162, 184, .5);
+            }
+            80%{
+                box-shadow: 0 0 0 10px rgba(23, 162, 184, 0), 0 0 0 6px rgba(23, 162, 184, 0);
+            }
+            100%{
+                box-shadow: 0 0 0 0 rgba(23, 162, 184, 0), 0 0 0 6px rgba(23, 162, 184, 0);
+            }
+        }
```

Comment:
So much wow !
But the bottom box shadow seems more translucent than the rest.

Reply:
As a dog in a spaceship would say: I had no idea what I was doing !

Thankfully It ended up looking nice but I will check if there is something to change